### ISSUE TITLE
Wrong information on a comment of PostRepository: method sanitizeSearchQuery()

### ISSUE THREAD

**Issue by o-alquimista** [symfony/demo#1023]
The DocBlock of `sanitizeSearchQuery()` says it is removing non-alphanumeric characters, while it seems to be only replacing duplicated spaces with a single space.

```
    /**
     * Removes all non-alphanumeric characters except whitespaces.
     */
    private function sanitizeSearchQuery(string $query): string
    {
        return trim(preg_replace('/[[:space:]]+/', ' ', $query));
    }
```

Someone forgot to implement the functionality? Or maybe forgot to remove the comment after the code it is about was removed.

**Comment by voronkovich** [comment]
@o-alquimista, You're right. See this commit: https://github.com/symfony/demo/commit/0e1c9d31ccb0b1eaa9c094f6716b6f5a5bff4297. I suggest to delete the `sanitizeSearchQuery` method and move it's code to the `extractSearchTerms` method. Could you please create a PR?

**Comment by o-alquimista** [comment]
@voronkovich It is done: https://github.com/symfony/demo/pull/1025, but the CI check failed. Why?

**Comment by javiereguiluz** [comment]
Fixed by #1025. Thanks!
